DQMOffline/Trigger/python/TopMonitor_cfi.py

```diff
@@ -59,7 +59,7 @@
 #pt binning
 hltTOPmonitoring.histoPSet.elePtBinning = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
 hltTOPmonitoring.histoPSet.jetPtBinning = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
-hltTOPmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
+ltTOPmonitoring.histoPSet.muPtBinning  = cms.vdouble(0,5,10,20,30,40,50,70,100,200,400)
```

Comment:
@parbol mind that you introduced a typo here. maybe you can check-out the file from the branch to make sure everything is the same as before for Top modules